server/validation/isPastDate.ts

```diff
@@ -0,0 +1,7 @@
+import dayjs from 'dayjs'
+
+export default function validate(value, options): boolean {
+  const today = dayjs().startOf('date')
+  const compare = dayjs(options, 'YYYY-MM-DD', true).startOf('date')
+  return compare.isSameOrBefore(today)
```

Comment:
I might be missing how this works but where is the `value` param actually used in the validation comparison?

Reply:
I use https://www.npmjs.com/package/validatorjs for validation, you need to set a validation rule against a field so for isPastDate I set it to the day field, I don't actually need the value of the day field but the whole date so pass that in as the options which can been seen https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/e60e3431e451eeaf918e925a0111607b9c851ac3/server/services/offenceService.ts#L25 and eventually setting it against the field https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/e60e3431e451eeaf918e925a0111607b9c851ac3/server/services/offenceService.ts#L43

Reply:
Ah that makes sense, thanks.